tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -317,6 +317,12 @@ internal static partial class ConfigurationKeys
         /// </summary>
         public const string PropagationStyleExtract = "DD_PROPAGATION_STYLE_EXTRACT";
 
+        /// <summary>
+        /// Configuration key for enabling automatic instrumentation on specified methods.
+        /// Default value is "" (disabled).
+        /// </summary>
+        public const string TraceMethods = "DD_TRACE_METHODS";
```

Comment:
I'm guessing this was a case of "Java did it first" and there was no RFC? If so, should we spend a few minutes and try to come up with a better name here, or is this already the convention across tracers? Seeing as `DD_TRACE` is the standard prefix, a setting called simply `METHODS` doesn't say much.

Reply:
The feature in its entirety is based on the `DD_TRACE_METHODS` / `@Trace` annotation work in Java. We could create a more usable name for the feature, but I think `DD_TRACE_METHODS` should at least be a fallback so if someone is running both Java and .NET applications they could set the same variable in both applications and get similar results

Reply:
> I'm guessing this was a case of "Java did it first" and there was no RFC? 
💯

Reply:
Ok, sounds good to me. If all the tracer later decide on a new name for this, we can keep this one as fallback. Thanks.